### PR TITLE
Add signed add-on capability and adjust max rating

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

https://github.com/home-assistant/supervisor/pull/3450#issuecomment-1041846447

Screenshot from 2022-02-16 18-00-49
Screenshot from 2022-02-16 18-00-56

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by bramkragten — DISMISSED** [review]

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 1]
```diff
@@ -364,6 +369,16 @@ class HassioAddonInfo extends LitElement {
...
+              ? html`
+                  <ha-chip hasIcon @click=${this._showMoreInfo} id="signed">
+                    <ha-svg-icon slot="icon" .path=${mdiLinkLock}></ha-svg-icon>
+                    ${this.supervisor.localize(
+                      "addon.dashboard.capability.label.signed"
+                    )}
+                  </ha-chip>
+                `
+              : ""}
           </div>
```
Comment (balloob) [inline hassio/src/addon-view/info/hassio-addon-info.ts:382]: Should this be a `<ha-chips>`?

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 2]
```diff
@@ -234,8 +239,8 @@ class HassioAddonInfo extends LitElement {
             <ha-chip
               hasIcon
               class=${classMap({
-                green: [5, 6].includes(Number(this.addon.rating)),
-                yellow: [3, 4].includes(Number(this.addon.rating)),
+                green: [6, 7, 8].includes(Number(this.addon.rating)),
+                yellow: [3, 4, 5].includes(Number(this.addon.rating)),
```
Comment (balloob) [inline hassio/src/addon-view/info/hassio-addon-info.ts:243]: This yellow could be a bit darker. It's hard to read with white text.

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 3]
```diff
@@ -234,8 +239,8 @@ class HassioAddonInfo extends LitElement {
...
                 red: [1, 2].includes(Number(this.addon.rating)),
```
[the remark this answers is not in the document]
Comment (balloob) [reply hassio/src/addon-view/info/hassio-addon-info.ts:244]: it's fine.

File: hassio/src/addon-view/info/hassio-addon-info.ts [hunk 4]
```diff
@@ -234,8 +239,8 @@ class HassioAddonInfo extends LitElement {
             <ha-chip
               hasIcon
               class=${classMap({
-                green: [5, 6].includes(Number(this.addon.rating)),
-                yellow: [3, 4].includes(Number(this.addon.rating)),
+                green: [6, 7, 8].includes(Number(this.addon.rating)),
```
Comment (balloob) [inline hassio/src/addon-view/info/hassio-addon-info.ts:242]:
```suggestion
                green: this.addon.rating >= 6,
```

File: hassio/src/addon-view/info/hassio-addon-info.ts (hunk 2 above)
Comment (ludeeus) [reply hassio/src/addon-view/info/hassio-addon-info.ts:243]: It's using this 
https://github.com/home-assistant/frontend/blob/3a664d45a943cd726dc578aa5dbc4707ad6e27d8/src/resources/ha-style.ts#L53-L58

Comment (balloob) [reply hassio/src/addon-view/info/hassio-addon-info.ts:244]: alright let's keep it for now.

**Review by balloob — DISMISSED** [review]

**Review by bramkragten — APPROVED** [review]

**Pull request merged**